### DISCUSSION TITLE
[Git] Upstream, Origin, 그리고 Local의 상호작용

### DISCUSSION THREAD

**Discussion by pykido** [BDD-CLUB/challenge-every-week-posting#409] [3. CS(컴퓨터 공학) 관련 글 모음]
[블로그 글 바로가기](https://animoto1.tistory.com/entry/Git-Upstream-Origin-%EA%B7%B8%EB%A6%AC%EA%B3%A0-Local%EC%9D%98-%EC%83%81%ED%98%B8%EC%9E%91%EC%9A%A9)
- 초록스터디하면서 배우게 된 fork, clone, pull request 등 다양한 git 개념들을 정리해봤습니다!!

**Comment by stopmin** [comment]
나중에 cherry pick 같은것도 쓸 일이 생긴다면 써보고  알려주세욤!! git stash도 알고싶숩니다..
깃 알면 알수록 모르겠숩니다...

**Reply by pykido** [reply]
넵 알겠숩니다!
